03_Day_Booleans_operators_date/03_booleans_operators_date.md

```diff
@@ -64,7 +64,7 @@ let isLightOn = true
 let isRaining = false
 let isHungry = false
 let isMarried = true
-let truValue = 4 > 3    // true
```

Comment:
name of a variable can be anything, so it doesn't need to be 'gramatically perfect' till it conveys the notion

Reply:
I agree that the name of a variable can be anything and doesn't need to be grammatically perfect as long as it conveys the intended meaning. In this case, the variable names like isLightOn, isRaining, isHungry, and isMarried make sense in terms of their purpose. It's important to prioritize clarity and readability in code.